src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -680,7 +680,8 @@ public void testCacheAndCheckWhichAddsNewResourceLocationButKeepsSameCheckerInst
 
         final DynamicalResourceHolderCheck check = new DynamicalResourceHolderCheck();
         final String firstExternalResourceLocation = getPath("checks" + File.separator
-            + "imports" + File.separator + "import-control_one.xml");
+            + "imports" + File.separator + "importcontrol"
+            + File.separator + "InputImportControlOne.xml");
```

Comment:
Checker test is located at:
src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java
so its inputs should be (please create folder):
src/test/resources/com/puppycrawl/tools/checkstyle/checker